diff --git a/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js b/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js
index f641a23..30abe88 100644
--- a/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js
+++ b/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js
@@ -6204,7 +6204,7 @@ function baseCreateRenderer(options, createHydrationFns) {
         // For fragments, directly remove all contained DOM nodes.
         // (fragment child nodes cannot have transition)
         let next;
-        while (cur !== end) {
+        while (cur && cur !== end) {
             next = hostNextSibling(cur);
             hostRemove(cur);
             cur = next;
